ntbs-service/Pages/Help/Menu.cshtml

```diff
@@ -15,7 +15,7 @@
     <h2 class="nhsuk-heading-l"> Help </h2>
 
     <div>
-        <a class="submenu-list-item-non-js" href="/Help">
+        <a class="submenu-list-item-non-js" href="@links.SharePointFAQPage" target="_blank" rel="noopener noreferrer">
```

Comment:
I didn't even know this page existed!

Do you know if it is linked to from anywhere on the site? Perhaps if you click on the "Help" option in the nav bar with javascript disabled?

Reply:
(I've checked and my suspicion was correct, so there's no action here)